Plugin/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "citation-plugin",
+  "version": "0.0.1",
+  "description": "A browser based tool for generating citations",
+  "main": "citation-plugin.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "install": "./node_modules/browserify/bin/cmd.js citation-plugin.js -o index.js",
+    "build": "./node_modules/browserify/bin/cmd.js citation-plugin.js -o index.js"
+    
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozillascience/citation-plugin.git"
+  },
+  "keywords": [
+    "citation"
+  ],
+  "author": "Samuel Mosher",
```

Comment:
Please add authors of the project and their emails. Should also include the mozilla user. An example can be found [here](https://github.com/mozillascience/CitationCore/blob/master/package.json).